modules/core/04-channel/keeper/grpc_query.go

```diff
@@ -538,6 +539,29 @@ func (q Keeper) UpgradeErrorReceipt(c context.Context, req *types.QueryUpgradeEr
 	return types.NewQueryUpgradeErrorResponse(receipt, nil, selfHeight), nil
 }
 
+// Upgrade implements the Query/UpgradeSequence gRPC method
+func (q Keeper) Upgrade(c context.Context, req *types.QueryUpgradeRequest) (*types.QueryUpgradeResponse, error) {
```

Comment:
```suggestion
func (k Keeper) Upgrade(c context.Context, req *types.QueryUpgradeRequest) (*types.QueryUpgradeResponse, error) {
```

Reply:
the linter disagreed with this change, the other methods all use `q` as the receiver

Reply:
I guess we will update it when we merge main